internal/use_cases/create_pull_request.go

```diff
@@ -43,8 +48,50 @@ type CreatePullRequest struct {
 	BranchProvider          domain.BranchProvider
 }
 
+func (cpr *CreatePullRequest) normalizeTemplatePath(path string) (string, error) {
+	if path == "" {
+		return "", nil
```

Comment:
suggestion: return a custom error if no template is provided.

Example error:
```go
var ErrNoTemplateProvided error = errors.New("no template path provided)
```
```suggestion
		return "", ErrNotemplateProvided
```

Note: we can simply return the error directly instead of creating a custom error for now.
```suggestion
		return "", errors.New("no template path provided")
```